src/Microsoft.Restier.Providers.EntityFramework/Submit/ChangeSetInitializer.cs

```diff
@@ -220,7 +220,7 @@ private void SetValues(DbEntityEntry dbEntry, DataModificationItem item, Type re
                                 propertyPair.Key));
                         }
 
-                        value = Activator.CreateInstance(type);
+                        value = propertyEntry.CurrentValue;
                         SetValues(value, type, dic);
```

Comment:
I am OK for this changes.
But add a TODO to line 257, the SetValues method is called by both patch and patch, we need to consider complex property has other complex type property cases.

Open an issue to track this and add to that TODO.